handel/src/services/apigateway/proxy/apigateway-proxy-template.yml

```diff
@@ -152,6 +152,35 @@ Resources:
           DNSName: !GetAtt CustomDomain{{logicalId name}}.DistributionDomainName
   {{/each}}
 
+  {{#if warmup}}
+  LambdaWarmupPermission:
+    Type: "AWS::Lambda::Permission"
+    Properties:
+      Action: 'lambda:InvokeFunction'
+      FunctionName: !Ref LambdaFunction
+      Principal: events.amazonaws.com
```

Comment:
Can this have a resource placed on it? I know some don't, but I thought I remembered that these can be constrained to the individual event defined in the template.